packages/ZbayLite/src/renderer/containers/widgets/createUsernameModal/CreateUsername.tsx

```diff
@@ -33,9 +34,54 @@ const CreateUsernameModal = () => {
   const joinCommunityModal = useModal(ModalName.joinCommunityModal)
   const createCommunityModal = useModal(ModalName.createCommunityModal)
   const loadingCommunityModal = useModal(ModalName.loadingPanel)
+  const unregisteredCommunitiesWithoutUserIdentity = useSelector(identity.selectors.unregisteredCommunitiesWithoutUserIdentity)
```

Comment:
What `unregisteredCommunitiesWithoutUserIdentity` means? It's hard to understand it on a higher level - I mean in a month you will not remember why you check `unregisteredCommunitiesWithoutUserIdentity.length`.
Could you add the boolean variable below with some readable meaning and use it in conditional?